client/src/components/Charts/Charts.jsx

```diff
@@ -218,7 +234,8 @@ class Charts extends React.Component {
                           {this.state.popup_data.map((row) => (
                             <TableRow key={row.id}>
                               <TableCell width="10%">{row.date}</TableCell>
-                              <TableCell width="10%">{row.state}</TableCell>
+                              {/* Change the following below to include {___} County, {state}?*/}
```

Comment:
I think the header 'County' should be clear enough for the reader, but potential question for the team